app/scripts/models/account.js

```diff
@@ -713,24 +713,43 @@ define(function (require, exports, module) {
     /**
      * Fetch the account's device list and populate the `devices` collection.
      *
-     * @param {object} devices - Devices collection
+     * @param {object} clients - Devices collection
```

Comment:
Is this same `clients` collection in `fetchOAuthApps`? Should they have a similar comment?